### ISSUE TITLE
Design and accessiblity review: II Beam Edit records, view nameservers, enroll domain

### ISSUE THREAD

**Issue by kimallen** [cisagov/manage.get.gov#4727]
### Sub-issue description

At this point in our DNS prototype, we can now edit DNS records, view the assigned nameservers, and an analyst can enroll a domain into DNS Hosting. 

The related features and functionality have already been reviewed by engineers in isolation and now we need designers to review them, as well as an integrated whole.

Be sure to address accessibility issues and to do a thorough a11y check.

### Acceptance criteria

- [ ] Design and accessibility review the DNS record management on the DNS records page
- [ ]  Document any changes needed in the Slack channel using our established norms (see Slack channel instructions)
- [ ]  Discuss changes with dev and determine which are small enough to be done as part of one collective ticket
- [ ] Work with dev to create new tickets for larger changes

### Additional context

Working with dev: Dev effort on this should be scoped to a "2" ticket

Dev is responsible for answering questions from design, making minor changes (such as those that we would normally make on a PR).
Anything that would take more than ~1 hours to complete alone should be a new issue.
If the changes desired will result in the dev spending more than 4 hours on all changes, make a dev ticket for items where reasonable (communicate with dev leads).

**Comment by kimallen** [comment]
Let's have a designer (or PM?) outside of the pod participate in this with fresh eyes

**Comment by katypies** [comment]
Marking this as ready since it looks like it's sized and ACs are all in place. @gabo0oo , if there's more we need to do before you start, we can discuss!

**Comment by Katherine-MN** [comment]
@gabo0oo Let's touch base on the size of this ticket.

**Comment by gabo0oo** [comment]
@Katherine-MN updating this to a 3.

**Comment by gabo0oo** [comment]
@katypies We will be creating a channel for this, right?

**Comment by gabo0oo** [comment]
> [@katypies](https://github.com/katypies) We will be creating a channel for this, right?

Nevermind, I see the channel! <3

**Comment by gabo0oo** [comment]
@kimallen @katypies Which engineer should be collaborating with us on this review?

**Comment by kimallen** [comment]
> [@kimallen](https://github.com/kimallen) [@katypies](https://github.com/katypies) Which engineer should be collaborating with us on this review?

@gabo0oo @asaki222 will be your dev.`olympic` will be your sandbox. I don't know if you'll need a separate channel- note that the intention is not to have the dev fix as you find things, but to support with ticket-writing as needed for issues found. (summary from [this thread](https://cisa-corp.slack.com/archives/C074E2QJ7J9/p1780007308233059))

**Comment by gabo0oo** [comment]
List of tickets created as part of this design review

For dev
[#5092 -Form field char limit updates](https://github.com/cisagov/manage.get.gov/issues/5092)
[#5092 - Dns Records Page: Layout broken at smaller viewports](https://github.com/cisagov/manage.get.gov/issues/5107)
[#5108 - Update "content" error message for CNAME, MX, and PTR records](https://github.com/cisagov/manage.get.gov/issues/5108)
[#5041 - MX record priority required error message appears twice](https://github.com/cisagov/manage.get.gov/issues/5041) (existing ticket)
[#4712 - Handle unsaved data when user changes record type in Add record form](https://github.com/cisagov/manage.get.gov/issues/4712) (existing ticket)

For design
[#5117 - Error message updates for Name field (A, AAAA, MX, CNAME, TXT)](https://github.com/cisagov/manage.get.gov/issues/5117)
[#5118 - Clarify character limits on Name field](https://github.com/cisagov/manage.get.gov/issues/5118)
[#5126 - Design: Update Domain overview page to support DNS hosting MVP](https://github.com/cisagov/manage.get.gov/issues/5126)

We also did a mini retro on how the review went, [that can be read here
](https://docs.google.com/document/d/1aPvxBlhXoWay09QEdDJNkY2KvWdk8z4AjNWBPmX_h4w/edit?tab=t.7f6b3u8l9vrd)
Closing this issue as done
